### DISCUSSION TITLE
How to write "default" in vsc.coverpoint ?

### DISCUSSION THREAD

**Discussion by jiyong00** [fvutils/pyvsc#185] [Q&A]
Does anyone know how to write "default" as vsc.coverpoint in systemverilog style coverpoint as follows?
```
    zero_delay : coverpoint cov_delay {
      bins ZERO = {1};
      bins MORE = default;  
    }
```

**Comment by mballance** [comment]
@jiyong00,
    Unfortunately, PyVSC does not currently support default bins. For now, you'll need to explicitly define a bin to represent what should go in a default bin.
    I agree that PyVSC should support default bins. I will consider implementing this enhancement.

Best Regards,
Matthew

**Reply by jiyong00** [reply]
Thank you for your reply,
I'm always grateful for your contributions.
